packages/server/src/admin/super.ts

```diff
@@ -27,6 +29,21 @@ superAdminRouter.post(
       return;
     }
 
+    const prefer = _req.header('Prefer');
+
+    if (prefer === 'respond-async') {
+      const { baseUrl } = getConfig();
+      const repo = res.locals.repo as Repository;
+      const exec = new AsyncJobExecutor(repo);
+      const job = await exec.start(_req.protocol + '://' + _req.get('host') + _req.originalUrl);
+      exec
```

Comment:
Can we add a `exec.startJob` method, which is not `async` and does not return a `Promise`, but does exactly this?  That would avoid the duplicated 4-lines in each of the handlers here.